ports/libopenmpt/CMakeLists.txt

```
@@ -11,21 +11,11 @@ endif()
 
 find_package(ZLIB REQUIRED)
 
-find_path(MPG123_INCLUDE_DIR NAMES mpg123.h)
-find_library(MPG123_LIBRARY  NAMES mpg123 libmpg123)
+find_package(MPG123)
+find_package(Vorbis)
+find_package(FLAC)
```

```suggestion
find_package(MPG123 REQUIRED)
find_package(Vorbis REQUIRED)
find_package(FLAC REQUIRED)
```
Since these ports are the dependencies of `mpg123`.

---

Hmm, thery are listed in manifest, so these packages will always be installed before libopenmpt build. Does it matter for unofficial CMake project?